src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,251 @@
+package Duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import Duke.Exceptions.*;
+import Duke.Task.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Duke {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int totalTasks = 0;
+
+    private static final String home = System.getProperty("user.home");
+    private static final String dirPath = home + File.separator + "Documents" + File.separator + "CS2113T";
+    private static final String filePath = home + File.separator + "Documents" + File.separator + "CS2113T"
+            + File.separator + "tasks.txt";
+
+    private static void loadFile() {
+        if(!new File(dirPath).exists()) {
+            System.out.println("Folder does not exist yet.");
+            File dir = new File(dirPath);
+            if(dir.mkdir()) {
+                System.out.println("New folder is created at this location: " + dirPath);
+            } else {
+                System.out.println("Oops! There's some problem in creating new folder.");
+            }
+        }
+        try {
+            readFileContents();
+        } catch (FileNotFoundException e) {
+            System.out.println("File is not found.");
+        }
+    }
+
+    private static void readFileContents() throws FileNotFoundException {
+        File f = new File(filePath);
+        try {
+            if(f.createNewFile()) {
+                System.out.println("New file is created in the folder for storing task list.");
+            }
+        } catch(IOException e) {
+            System.out.println("An error has occurred.");
+        }
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String thisLine = s.nextLine();
+            String[] words = thisLine.split("\\|", 4);
+            switch(words[0].trim()) {
+            case "T":
+                tasks.add(new Todo(words[2].trim()));
+                break;
+            case "E":
+                tasks.add(new Event(words[2].trim(), words[3].trim()));
+                break;
+            case "D":
+                tasks.add(new Deadline(words[2].trim(), words[3].trim()));
+                break;
+            default:
+                System.out.println("This line is not added to list:" + thisLine);
+            }
+            if(Integer.parseInt(words[1].trim())==1) {
```

Comment:
I think there's inconsistent spacing here. I believe it should be `(words[1].trim()) == 1`